### PR TITLE
fix so that dygraph is actually available from outside

### PR DESCRIPTION
**Pull request by timelyportfolio**
This should resolve https://github.com/rstudio/dygraphs/issues/196.

### CODE REVIEW

File: inst/htmlwidgets/dygraphs.js [hunk 1]
```diff
@@ -777,7 +777,7 @@ HTMLWidgets.widget({
       },
       
       // export dygraph so other code can get a hold of it
-      dygraph: dygraph
+      dygraph: null
```
Comment (pshevtsov) [inline inst/htmlwidgets/dygraphs.js]: Why exporting `dygraph` as `null`?

Comment (timelyportfolio) [reply inst/htmlwidgets/dygraphs.js]: not exporting as `null` rather intialiazing as `null` which gets populated on render in https://github.com/timelyportfolio/dygraphs_htmlwidget/blob/b1834d8608c011d210e79cfedcd259e27efcec20/inst/htmlwidgets/dygraphs.js#L231.  I should note a function version would work here if you prefer.  This is similar to leaflet's `getMap()` ([lines](https://github.com/rstudio/leaflet/blob/master/inst/htmlwidgets/leaflet.js#L518-L521)) but might surprise dygraphs users expecting an `object`.

```
dygraph: function() {return dygraph;}
```

**Comment by timelyportfolio** [comment]
Bummed this didn't make it to CRAN. Any additional feedback on this?  Without it, we cannot modify with something like `htmlwidgets::onRender`.  Thanks.

**Comment by pshevtsov** [comment]
I'll have a look tomorrow. If everything is OK, we'll do a minor release.

**Comment by timelyportfolio** [comment]
@pshevtsov, looks like I am potentially back on the `dygraphs` train after about a year absence.  Hope to be able to make some more contributions.  Thanks for all your work!

**Pull request merged**